docs/manuals/gui/pages/html.md

```diff
@@ -16,12 +16,103 @@ html_page = Html("""
 content can be used here.
+
+<taipy:button>Press</taipy:button>
 """)
 ```
 
-The *html_page* variable contains the definition of a page whose content is defined from HTML text.
+The *html_page* variable holds a page whose content is defined from HTML text.
 
 Taipy identifies visual element definitions by finding tags that belong to the `taipy` namespace.
 You can find details on how to create visual elements using HTML in the
 [HTML Syntax](../viselements/index.md#html) section about Visual Elements definition.
 
+# XHTML is required
+
+When Taipy GUI renders the page, it parses the text content to locate the potential visual element
+tags and convert them into new HTML code.<br/>
+This process requires the HTML content to be expressed as [XHTML](https://www.w3.org/TR/xhtml1/).
+XHTML (which stands for "Extensible Hypertext Markup Language") is a markup language that mirrors
+HTML but adheres to the stricter syntax rules of XML (Extensible Markup Language).
+
+Here are the rules that the page definition must follow:
+- The HTML content must be a well-formed XML document (although it does not need a root
+  element).<br/>
+    - Every opening tag must be explicitly closed.<br/>
+      For example, although HTML accepts the tag `<img src="...">`, XHTML does not. You must
+      indicate where the `img` tag ends.<br/>
+      That can be `<img src="..."/>` or `<img src="..."></img>`.
+    - 
+- Strict adherence to syntax rules is required. All elements must be properly nested, closed, and in
+  lowercase.
+    - All the tag names must be lowercase.
+    - All the attribute names must be lowercase, and all values must be quoted.<br/>
+      Note that the XHTML syntax parsed by Taipy GUI in the context of a visual element allows for
+      leaving an attribute valueless. The value for the property is then set to True.
+
+# Event handlers
+
+Because of Cross Site Scripting (XSS) concerns, when Taipy GUI interprets the HTML content, it will
+drop the event handlers set on native HTML elements.
+
+Here is an example of such a use case.<br/>
+This is a Taipy application that creates a page entirely defined in HTML:
+```python linenums="1"
+from taipy.gui import Gui, Html
+
+page = Html("""
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Invoking JavaScript</title>
+    <script type="text/javascript">
+        function send_alert() {
+            alert("Just got an alert!")
+        }
+    </script>
+</head>
+<body>
+    <h1>Sending an alert</h1>
+    <button id="alert_button" onclick="send_alert();">Send alert</button>
+</body>
+</html>
+""")
+
+Gui(page).run()
+```
+
+In line 16, you can spot where the HTML button is created, along with its event handler: the
+function *send_the_alert()*, defined in line 9, is called when the user presses the button.
+
+But if you run this application and press the button, you will *not* witness the invocation of the
+function. That is because when Taipy converts the HTML input into the page sent to the browser, it
```

Comment:
this is not true
if you look at the browser's console you'll see that
- onclick => onClick
- onClick="send_alert();" => jsxParser complains that that property should be a function and not a string

if you write it like this onClick="{!send_alert!}", the parser does not complain (that's the way we use to pass js objects to jsxParser.

but ... it's not working with nothing in the console

Reply:
I'm trying to not expose users to JSX.
Note how the term "React" appears basically nowhere (except in the Extension API of course).